Makefile

```diff
@@ -60,6 +65,16 @@ cephcsi:
 	if [ ! -d ./vendor ]; then (go mod tidy && go mod vendor); fi
 	GOOS=linux GOARCH=$(GOARCH) go build -mod vendor -a -ldflags '$(LDFLAGS)' -o _output/cephcsi ./cmd/
 
+.PHONY: containerized-build
+containerized-build: .devel-container-id
+	$(CONTAINER_CMD) run --rm -v $(PWD):/go/src/github.com/ceph/ceph-csi$(SELINUX_VOL_FLAG) $(CSI_IMAGE_NAME):devel make -C /go/src/github.com/ceph/ceph-csi cephcsi
```

Comment:
No, that would cause conflicts as there is a version to be published, and we do not need any of the development tools in there.

Reply:
With new dockerfile we will have 2 dockerfiles, I would like to merge 2 dockerfiles into one and use multistage docker build

Reply:
That can be done, but I do not think it needs to be in this PR. With this change, the build with `make containerized-build` provides the executable under `_output` already. Adjusting the `Makefile` to consume that binary in the `image-cephcsi` target could be an alternative to multistage builds.

If you really are interested in multi-stage builds, I can think about that too. With #890 I am already doing more containerized things. Just create a new Issue for multi-stage builds and assign it to me, so that I can combine/prepare the work.